gapir/cc/interpreter.cpp

```diff
@@ -83,6 +90,11 @@ bool Interpreter::run(const uint32_t* instructions, uint32_t count) {
     GAPID_ASSERT(mCurrentInstruction == 0);
     mInstructions = instructions;
     mInstructionCount = count;
+    mCrashHandler.registerHandler([this](const std::string& minidumpPath, bool succeeded) {
+        GAPID_ERROR("--- CRASH DURING REPLAY ---");
+        GAPID_ERROR("LAST COMMAND:     %d", mLabel);
+        GAPID_ERROR("LAST INSTRUCTION: %d", mCurrentInstruction);
+    });
```

Comment:
Not capturing the Unregister lambda here, so this handler never gets removed. The Interpreter will get destructed but the crash handler doesn't (until the end of the program at least), so `this->mLabel` will cause a crash within a crash.